README.md

```diff
@@ -4,19 +4,33 @@
 
 ## Time — Clock
 
-> Not really useful, but every dashboard composition tool must have one :)
+Show analog clock from local or anoter timezone.
 
 ![clock](https://raw.githubusercontent.com/plouc/mozaik-ext-time/master/preview/time.clock.png)
 
 ### parameters
 
-*This widget has no parameter*
+Both parameters are optional:
```

Comment:
You should use the same format used on other extensions for parameters (https://github.com/plouc/mozaik-ext-jenkins/blob/master/README.md#parameters)